src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1226,6 +1226,23 @@ public override void VisitSimpleAssignmentExpression(ISimpleAssignmentExpression
             Visit(operation.Value, "Right");
         }
 
+        public override void VisitDeconstructionAssignmentExpression(IDeconstructionAssignmentExpression operation)
+        {
+            LogString(nameof(IDeconstructionAssignmentExpression));
+            LogCommonPropertiesAndNewLine(operation);
+
+            Visit(operation.Target, "Left");
+            Visit(operation.Value, "Right");
```

Comment:
Why use different names than the properties?

Reply:
I matched the dumper output with ISimpleAssignmentExpression, which also seems to be using Left and Right.